src/demo-app/accessibility/accessibility.html

```diff
@@ -0,0 +1,36 @@
+<md-sidenav-container class="demo-root" fullscreen>
```

Comment:
For the a11y demo, I think it would be better to omit the sidenav and just iterate over the `<a>` elements inside a `<nav>` at the top of the page. Not pretty, but simple. The sidenav will potentially just complicate the a11y testing, since we want each page to be totally focused on the component under test.

We can also consider adding a [skip link](http://webaim.org/techniques/skipnav/) at the beginning